examples/xml-io/xml_io.out

```diff
@@ -1,6 +1,5 @@
 # To run this sample, navigate to the directory that contains the
-# `.bal` file, and execute the `bal run` command below.
+# `.bal` file, and execute the `ballerina run` command below.
```

Comment:
```suggestion
# `.bal` file, and execute the `bal run` command below.
```

Reply:
Fixed in here https://github.com/ballerina-platform/ballerina-distribution/pull/1174/commits/6361926fb645bcaed2658e294c8e1402ec57d577